layouts/partner/aws.html

```diff
@@ -76,42 +75,38 @@ <h2>Creating a Serverless REST API</h2>
                         This example shows how to create a simple REST API that counts the number of times a route has been hit. To implement this API, we need a DynamoDB table, an API endpoint, and a Lambda function.
                     </p>
                     <p>
-                        <a class="btn" href="{{ relref . "/docs/quickstart" }}">Get Started</a>
+                        <a class="btn inline-block my-2" href="{{ relref . "/docs/quickstart" }}">Get Started</a>
                     </p>
                     <p>
                         Pulumi can be used on any resource covering serverless, containers, and infrastructure using JavaScript, TypeScript, Python, and Go.
                     </p>
                 </div>
                 <div class="md:w-1/2 md:ml:4">
-                    <div class="highlight">
-
-<pre><code>const cloud = require("@pulumi/cloud-aws");
+                    {{ $code := `const cloud = require("@pulumi/cloud-aws");
 
-// Create a mapping from 'route' to a count
+// Create a mapping from 'route' to a count.
 let counterTable = new cloud.Table("counterTable", "route");
 
-// Create an API endpoint
+// Create an API endpoint.
 let endpoint = new cloud.API("hello-world");
 
 endpoint.get("/{route+}", (req, res) => {
     let route = req.params["route"];
-    console.log(`Getting count for '${route}'`);
+    console.log("Getting count for " + route);
```

Comment:
Bummer. Too bad there isn't a way to easily escape backticks.

Reply:
Yeah, it’s a huge bummer. I did notice, though, that the backticks in the code samples I put in frontmatter variables _are_ being rendered, so there might be a way to do this somehow. I also really hate having to force all the code to the left just to avoid leading space. So there’s still room for improvement here. I’ll keep messing with it. :)